mono/mini/interp/interp.h

```diff
@@ -42,4 +42,7 @@ typedef struct _InterpMethodArguments InterpMethodArguments;
  */
 MONO_API void mono_ee_interp_init (const char *);
 
+void
+mono_wasm_interp_to_native_trampoline (void *target_func, InterpMethodArguments *margs);
```

Comment:
#ifdef TARGET_WASM ?, for aesthetics ?

Reply:
I don't think it is worthwhile, but it is a style, yes. I'll do it.
But host, I think.
TARGET is for JIT. I think.
HOST is for runtime, metadata, gc. interp. I think.

Reply:
I'll submit another PR to fix some host/target mixups.
It seems to be often confused, and that ends up ok, because relatively few cross configurations are used.